### DISCUSSION TITLE
Is there any appium integration with flutter apps available with code magic?

### DISCUSSION THREAD

**Discussion by SAJILCH** [codemagic-ci-cd/codemagic-docs#2339] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Unable to find flutter appium integration docs

### Expected results

Support for flutter appium integration. 

### Actual results

Support for flutter appium integration is not found

**Comment by himesh-cm** [comment]
Hi @SAJILCH,
We do not currently have an integration guide available for using Appium with Flutter. However, there are many resources available on the web that can provide some guidance. You may find this reference helpful: https://github.com/appium/appium-flutter-driver

**Comment by himesh-cm** [comment]
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
